README.md

```diff
@@ -63,6 +63,10 @@ Please also check the documentation of [Language Support for Java by Red Hat](ht
   - `internalConsole` - VS Code debug console (input stream not supported).
   - `integratedTerminal` - VS Code integrated terminal.
   - `externalTerminal` - External terminal that can be configured in user settings.
+- `shortenCommandLine` - The specified command line style to launch the program. Defaults to `none`.
+  - `none` - Launch the program with the standard command line 'java [options] classname [args]'.
```

Comment:
Add 'auto' as the default value to unblock majority of use cases.

Reply:
A new PR https://github.com/Microsoft/java-debug/pull/254 on java-debug to help compute the launch command length.